WordPress/wordpress-playground
# Doc: Link to the Playground-tools repo in the main README

## What?


- Remove the mention to wp-now from the main README.

## Why?



The new repo is: https://github.com/WordPress/playground-tools
I also considered to leave the reference and point it to the new repo, but no other tools are shared in the main repo.

## Testing Instructions

- Check the new README on this branch: https://github.com/WordPress/wordpress-playground/blob/remove/wp-now-mention-on-main-readme/README.md
- Observe there are no mentions to wp-now.
